src/core/server/services/migrate/migrations/1569947670260_add_moderator_notes_to_user.ts

```diff
@@ -0,0 +1,49 @@
+import { Db } from "mongodb";
+
+import collections from "coral-server/services/mongodb/collections";
+
+import Migration from "coral-server/services/migrate/migration";
+
+export default class extends Migration {
+  public async up(mongo: Db, tenantID: string) {
+    const result = await collections.users(mongo).updateMany(
+      {
+        tenantID,
+        moderatorNotes: null,
+      },
+      {
+        $set: {
+          moderatorNotes: [],
+        },
+      }
+    );
+    this.log(tenantID).warn(
+      {
+        matchedCount: result.matchedCount,
+        modifiedCount: result.matchedCount,
+      },
+      "added empty moderatorNotes array"
+    );
+  }
+
+  public async down(mongo: Db, tenantID: string) {
+    const result = await collections.users(mongo).updateMany(
+      {
+        tenantID,
+        $exists: "moderatorNotes",
```

Comment:
The syntax for the `$exists` operator is actually:

```suggestion
        moderatorNotes: { $exists: true },
```